Author: Kilian Krause <kilian@debian.org>
Description: Fix directory naming of notrace OBJDIR to have just one 'n'
Index: h323plus/src/Makefile
===================================================================
--- h323plus.orig/src/Makefile
+++ h323plus/src/Makefile
@@ -41,10 +41,6 @@ include $(OPENH323DIR)/openh323u.mak
 STDCCFLAGS += -fno-strict-aliasing
 
 OH323_OBJDIR = $(OH323_LIBDIR)/$(PT_OBJBASE)
-ifdef NOTRACE
-OH323_OBJDIR := $(OH323_OBJDIR)n
-endif
-
 
 OBJDIR	=	$(OH323_OBJDIR)
 LIBDIR	=	$(OH323_LIBDIR)
